package.json

```diff
@@ -5,21 +5,30 @@
   "main": "index.js",
   "scripts": {
     "dev": "npm run generate && npm run start",
-    "generate": "npm run generate:nextjs && npm run generate:django",
+    "generate": "npm run generate:nextjs && npm run generate:django && npm run generate:flask && npm run generate:fastapi && npm run generate:starlette",
     "generate:nextjs": "node generator/nextjs/confirm.mjs && node generator/nextjs/generate.mjs",
     "generate:django": "node generator/django/confirm.mjs && node generator/django/generate.mjs",
+    "generate:flask": "node generator/flask/confirm.mjs && node generator/flask/generate.mjs",
+    "generate:fastapi": "node generator/fastapi/confirm.mjs && node generator/fastapi/generate.mjs",
+    "generate:starlette": "node generator/starlette/confirm.mjs && node generator/starlette/generate.mjs",
     "install-all": "concurrently -n w: npm:install:*",
     "install:nextjs:template": "npm i --prefix generator/nextjs/template",
     "install:affinidi-nextjs-nextauthjs": "npm i --prefix samples/affinidi-nextjs-nextauthjs",
     "install:auth0-nextjs-nextauthjs": "npm i --prefix samples/auth0-nextjs-nextauthjs",
-    "install:affinidi-django-authlib": "pip install -r samples/affinidi-django-authlib/requirements.txt",
+    "install:affinidi-django-authlib": "pip install -r samples/affinidi-django-authlib/requirements.txt",   
     "install:auth0-django-authlib": "pip install -r samples/auth0-django-authlib/requirements.txt",
+    "install:affinidi-flask-authlib": "pip install -r samples/affinidi-flask-authlib/requirements.txt",
+    "install:affinidi-fastapi-authlib": "pip install -r samples/affinidi-fastapi-authlib/requirements.txt",
+    "install:affinidi-starlette-authlib": "pip install -r samples/affinidi-starlette-authlib/requirements.txt",
     "start": "concurrently -n w: \"npm:start:*(!template)\"",
     "start:template": "npm run --prefix generator/template dev -- -p 3000",
     "start:affinidi-nextjs-nextauthjs": "npm run --prefix samples/affinidi-nextjs-nextauthjs dev -- -p 3001",
     "start:auth0-nextjs-nextauthjs": "npm run --prefix samples/auth0-nextjs-nextauthjs dev -- -p 3002",
     "start:affinidi-django-authlib": "python3 samples/affinidi-django-authlib/manage.py migrate && python3 samples/affinidi-django-authlib/manage.py runserver 8000",
     "start:auth0-django-authlib": "python3 samples/auth0-django-authlib/manage.py migrate && python3 samples/auth0-django-authlib/manage.py runserver 8001",
+    "start:affinidi-flask-authlib": "cd samples/affinidi-flask-authlib && python3 -m flask run",
```

Comment:
Can we provide different ports for all these apps to avoid collision of ports

Reply:
yes, I have updated ports for all reference application. 
FastAPI-8200
flask-8100
starlette-8300

Also updated Readme